XamarinAndroidChatClient/XamarinAndroidChatClient/MainActivity.cs

```diff
@@ -0,0 +1,92 @@
+﻿using Android.App;
+using Android.Widget;
+using Android.OS;
+using System;
+using System.Collections.Generic;
+using Microsoft.AspNetCore.SignalR.Client;
+
+namespace XamarinAndroidChatClient
+{
+    [Activity(Label = "Xamarin Android Chat Client", MainLauncher = true)]
+    public class MainActivity : Activity
+    {
+        protected override void OnCreate(Bundle savedInstanceState)
+        {
+            base.OnCreate(savedInstanceState);
+
+            // Set our view from the "main" layout resource
+            SetContentView(Resource.Layout.Main);
+
+            //Setting up references to views
+            var connectButton = FindViewById<Button>(Resource.Id.bConnect);
+            var listView = FindViewById<ListView>(Resource.Id.lvMessages);
+            var sendButton = FindViewById<Button>(Resource.Id.bSend);
+            var messageText = FindViewById<EditText>(Resource.Id.etMessageText);
+            var connected = false;
+
+            var hubConnection = new HubConnectionBuilder()
+                    .WithUrl("https://signalrcore-chatsample.azurewebsites.net/chat")
```

Comment:
Oh, I should remove the url.

Reply:
Also it looks like you can connect to localhost through an Android emulator. 
https://stackoverflow.com/questions/5528850/how-to-connect-localhost-in-android-emulator

Reply:
What is this designed to connect to? The ChatSample in this repo? Maybe throw a quick comment in for that :)

Reply:
>What is this designed to connect to? The ChatSample in this repo?

Yup, exactly 
I'll add a readme to explain how to get it running. 👍

Reply:
@mikaelm12 - because localhost is the phone itself and you are not running the server on the phone.

Reply:
Yeah, I remember how we did the SignalR 2.2.2 Xamarin verification and recreated that for this sample 😄 
Looks like we could have avoided that all along ha